Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,128 @@
+Parser:
+  Title: Network Session ASIM filtering parser for Cisco ISE
+  Version: '1.0'
+  LastUpdated: July 10, 2023
+Product:
+  Name: Cisco ISE
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.6'
+References:
+  - Title: ASIM Network Session Schema
+    Link: https://aka.ms/ASimNetworkSessionDoc
+  - Title: ASIM
+    Link: https://aka.ms/AboutASIM
+  - Title: Organization Application Security Events 
+    Link: https://www.cisco.com/c/en/us/td/docs/security/ise/3-2/admin_guide/b_ise_admin_3_2/b_ISE_admin_32_maintain_monitor.html#ID58
+Description: |
+  This ASIM parser supports normalizing Cisco ISE Events produced by the Microsoft Sentinel Cisco ISE connector to the ASIM Network Session normalized schema.
+ParserName: vimNetworkSessionCiscoISE                                 
+EquivalentBuiltInParser: _Im_NetworkSession_CiscoISE                                 
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: ipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstportnumber
+    Type: int
+    Default: int(null)
+  - Name: hostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvcaction
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
```

Comment:
@spsocprime Changes looks good. Please format query to re-organize code

Reply:
@spsocprime Can you please take a look?

Reply:
formatted query